### ISSUE TITLE
Receive midi messages from all connected devices simultaneously 

### ISSUE THREAD

**Issue by elmcapp** [djipco/webmidi#258]
Is there a way to receive midi messages from all connected midi devices simultaneously or do we still need to create a different listener for each midi devices.

**Comment by djipco** [comment]
No. Currently, you have to add a listener for all devices. Obviously, all those listeners can call the same function, though.

By the way, when you have questions such as this one, you should post to [GitHub Discussions](https://github.com/djipco/webmidi/discussions) instead of filing a new issue.
